<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Surefire suite" duration-ms="5589" started-at="2021-09-13T21:32:58 IST" finished-at="2021-09-13T21:33:03 IST">
    <groups>
    </groups>
    <test name="Surefire test" duration-ms="5589" started-at="2021-09-13T21:32:58 IST" finished-at="2021-09-13T21:33:03 IST">
      <class name="Karate.KarateScripts.APIRunnerTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:Karate.KarateScripts.APIRunnerTest@6895a785]" name="beforeTest" is-config="true" duration-ms="14" started-at="2021-09-13T21:32:58 IST" finished-at="2021-09-13T21:32:58 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="parallelRun()[pri:0, instance:Karate.KarateScripts.APIRunnerTest@6895a785]" name="parallelRun" duration-ms="5547" started-at="2021-09-13T21:32:58 IST" finished-at="2021-09-13T21:33:03 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[failed features:
resources.TwoWAyCommands: -unknown-:40 - 
TokenTwoWay.feature:21 - status code was: 429, expected: 200, response time: 1218, url: https://restdev1.tracking.thermoking.com/connected-solutions/obtainToken/, response: {"path":"/connected-solutions/obtainToken/","error":"Too Many Requests","message":"This client has been exceeded the allowance of requests for this API","timestamp":"2021-09-13T16:03:01.686+0000","status":429}
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: failed features:
resources.TwoWAyCommands: -unknown-:40 - 
TokenTwoWay.feature:21 - status code was: 429, expected: 200, response time: 1218, url: https://restdev1.tracking.thermoking.com/connected-solutions/obtainToken/, response: {"path":"/connected-solutions/obtainToken/","error":"Too Many Requests","message":"This client has been exceeded the allowance of requests for this API","timestamp":"2021-09-13T16:03:01.686+0000","status":429}

at org.junit.Assert.fail(Assert.java:88)
at org.junit.Assert.assertTrue(Assert.java:41)
at Karate.KarateScripts.APIRunnerTest.parallelRun(APIRunnerTest.java:54)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:136)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:145)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- parallelRun -->
      </class> <!-- Karate.KarateScripts.APIRunnerTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
